docs/governance-workbook/README.md

```diff
@@ -6,6 +6,125 @@
+    -	Virtual Machine List filtered by Power state
+
+## Storage + backup
+The **Storage + backup** tab is focused on storage and backup resources:
+- Count of all resource types
+- Resource details
+- Storage accounts details
+  - Overview
+  - Capacity
+- Backup details (Pre-requisite: Vault diagnostic setting needs configured with Log Analytics Workspaces)
+
+## Network
+The **Network** tab is focusing on Network resources configuration:
+- Count of all network resources by resource type
+-	**NSGs** is listing orphan Network Security Groups
+-	**NSG Rules** (if a NSG is selected above this list) is listing all Network Security Groups rules
+-	**Public Ips** is listing Public IPs (could be filtered if orphan or not)
+-	**Application Gateways** is listing Application Gateways with or without any backend IP and backend Addresses (depend on the “Orphan filter parameter”)
+-	**Load Balancers** is listing Load Balancers with or without empty backend pools  (depend on the “Orphan filter parameter”)
+
+## PaaS
+The **PaaS** tab is focusing PaaS resources configuration:
+-	Automation is listing Automation Accounts, LogicApp Connectors, LogicApp API, Connectors, Logic Apps, Automation Runbooks, Automation Configurations.
+-	App Services is listing App Service Plans, Azure Functions, API Apps, App Services, App Gateways, Front Door, API Management, App Certificates, App Config Stores
+-	Data is listing CosmosDB, SQL DBs, MySQL Servers, SQL Servers, PostgreSQL Servers, PostgreSQL Flexi Servers, MariaDB Servers.
+-	Storage is listing Azure File Sync, Azure Backup, Storage Accounts, Key Vaults
+
+## Security
+The **Security** tab is focusing on the security score for your subscriptions and controls
+- Security Scores by Subscription
+- Security Scores by Control
+- Top 5 attacked resources (with High Severity)
+- Top alert types
+- New Alerts (Since last 24hrs)
+- MITRE ATT&CK tactics
+- Active Alerts
+
+## Monitoring
+The **Monitoring** tab is providing Service Health information and main events that are happening into one selected subscription:
+-	All Service Health active Incident
+-	All changes performed on your resources for the past one day
+-	All deleted resources for the past 14 days
+
+## Service retirement
+The **Services retirement** tab shows Azure services that are being phased out so that you can mitigate affected resources
+
+## Resource age
+The **Resource age** tab is giving you more information about the resource “Creation Date” and the “Last Change Date” in the selected Subscription to help you to identify old resources and perform sanitization.
+
+## Tag explorer
+The **Tag explorer** tab help you to filter/sort your resources by Tag. You can list and identify resources with or without a specified tag name and with or without a value. Each result can be filtered by resource type.
+You can also get general information on Subscription and Resource Groups.
```

Comment:
Don't forget to add specing between paragraphs. Otherwise markdown will merge them together.
```suggestion

You can also get general information on subscriptions and resource groups.
```